### DISCUSSION TITLE
ICST 2025 Results Notification

### DISCUSSION THREAD

**Discussion by ChristianBirchler** [christianbirchler-org/sdc-testing-competition#37] [Announcements]
Dear tool competitors,

All tools submitted for the ICST 2025 SDC Tool Competition have been assessed and are eligible for further evaluation.

> NOTE: The results notification only confirms that the tools are assessed and passed a preliminary quality check for the final evaluation.

**Important:**
Please remember to email Mattia Fazzini, one of the co-chairs of the Self-Driving Car Testing Tool Competition.
We would like to collect information about all the authors of the tool.
Can you send the first name, surname, email, and affiliation for all the authors to [mfazzini@umn.edu](mailto:mfazzini@umn.edu)? 

You find the same notice in your Pull Requests.

We appreciate your help making this competition an excellent experience for all participants.

**Comment by ali-ihsan-gullu** [comment]
Hello.  Can we say that all the tools submitted have qualified for presentation at the conference in Italy and are required to submit a 2-page camera-ready paper?

**Reply by ChristianBirchler** [reply]
@mattiafazzini will give further information and details regarding the 2-page paper via email. All participants to the competition should ensure to share the contact details of all authors to @mattiafazzini (checkout the Pull Requests).

**Comment by ali-ihsan-gullu** [comment]
Thank you very much for your rapid responses. @mattiafazzini sent us the mentioned email.

**Comment by ali-ihsan-gullu** [comment]
Hello. 

Will the final comparison results be shared or will we be able to find out at the conference? 

Thank you so much.

**Reply by ChristianBirchler** [reply]
Hi @vatozZ, initially, we intended to present the report at the conference, but we assume you are interested in knowing how your tool performed with the benchmarks.

Overall, the ITS4SDC tool shows the best results in terms of cost-effectiveness. We share the reports (ICST and SBFT) in the v1.2.0 release: https://github.com/christianbirchler-org/sdc-testing-competition/releases/tag/v1.2.0

Looking forward to seeing your tool presentation at ICST!

Best regards,
Christian

**Reply by ali-ihsan-gullu** [reply]
Dear Christian, 

We were eagerly looking forward to the results, which is why I asked. We sincerely appreciate you sharing the report with us. Additionally, we would like to express our gratitude for promptly responding to all discussions on github and providing a trouble-free, excellent competition experience. 

Looking forward to seeing you in Naples !

Yours sincerely,
Ali

**Comment by ChristianBirchler** [comment]
Again, thank you all for participating in this competition. It was a pleasure to run all the tools without any issues for the evaluation. This would not have been possible with the great engineering work of all of you.

You find the competition reports as attachments in the latest release: https://github.com/christianbirchler-org/sdc-testing-competition/releases/tag/v1.2.0

See you hopefully all soon at the conference!

Best regards,
Christian
